.classpath

```diff
@@ -1,11 +1,11 @@
 <?xml version="1.0" encoding="UTF-8"?>
```

Comment:
The changes may need to be discarded or changed base on pr: https://github.com/eisopux/checker-framework-eclipse/pull/8